python/pylibcugraph/pylibcugraph/utils.pyx

```diff
@@ -18,6 +18,7 @@ from libc.stdint cimport uintptr_t
 
 import numpy
 import cupy
+import cudf
```

Comment:
There shouldn't be any cudf calls in pylibcugraph - was this an artifact of an old merge?

Reply:
Yes, from the first MG Hits merge. It's been removed in this next commit